modules/processing/cuckooml.py

```diff
@@ -15,11 +15,18 @@
 from lib.cuckoo.common.constants import CUCKOO_ROOT
 from math import log
 
+if Config("cuckooml").cuckooml.plotting:
+    try:
+        import matplotlib.pyplot as plt
+        import seaborn as sns
+    except ImportError, e:
+        print >> sys.stderr, "Some error while importing"
```

Comment:
Meaningful error message would be nice, e.g. "Plotting libraries (matplotlib and seaborn) are missing.".